test/docker.js

```diff
@@ -1,9 +1,64 @@
 const { exec } = require('child-process-promise');
+const utils = require('../src/modules/utils');
 
-const docker = 'docker';
-const containerName = 'arnavmq-tests';
+class Docker {
+  constructor() {
+    this.name = 'arnavmq-tests';
+  }
 
-module.exports.run = () => exec(`${docker} run --detach --name=${containerName} -p 5672:5672 rabbitmq:3.8-alpine`);
-module.exports.start = () => exec(`${docker} start ${containerName}`);
-module.exports.stop = () => exec(`${docker} stop ${containerName}`);
-module.exports.rm = () => exec(`${docker} rm --force --volumes ${containerName} || true`);
+  async start() {
+    // We might be already running, let's cleanup so can a have a fresh start ;)
+    await exec(`docker rm --force --volumes ${this.name} || true`);
```

Comment:
maybe better to have in the constructor, is there any chance `start` will be called twice or more during the tests?

Reply:
`start` is called only before all tests are running, we can't put this `docker rm` in constructor, since it's async operation